### ISSUE TITLE
Formatting MapList indent

### ISSUE THREAD

**Issue by HSGamer** [Carleslc/Simple-YAML#43]
My code:
```java
        Config config = new PluginConfig(this, "config.yml");

        if (config.getConfig().isSet("test-path"))
            config.getConfig().getMapList("test-path").forEach(map -> map.forEach((o, o2) -> System.out.println(o + ": " + o2)));

        List<Map<String, Object>> mapList = new ArrayList<>();

        Map<String, Object> map1 = new HashMap<>();
        map1.put("1", "test1");
        map1.put("2", "test2");
        map1.put("3", "test3");
        mapList.add(map1);
        Map<String, Object> map2 = new HashMap<>();
        map2.put("4", "test1");
        map2.put("5", "test2");
        map2.put("6", "test3");
        mapList.add(map2);
        Map<String, Object> map3 = new HashMap<>();
        map3.put("7", "test1");
        map3.put("8", "test2");
        map3.put("9", "test3");
        mapList.add(map3);
        config.getConfig().set("test-path", mapList);

        config.saveConfig();
```

* When running this code, the file will be saved like this
```yaml
test-path:
  -
  '1': test1
  '2': test2
  '3': test3
  -
  '4': test1
  '5': test2
  '6': test3
  -
  '7': test1
  '8': test2
  '9': test3

```
* Instead of
```yaml
test-path:
- '1': test1
  '2': test2
  '3': test3
- '4': test1
  '5': test2
  '6': test3
- '7': test1
  '8': test2
  '9': test3
```
* A simple fix for this is to remove [this line](https://github.com/Carleslc/Simple-YAML/blob/dd8c388f13af1a66ade4bbdb4ee6da60258e2f6c/Simple-Yaml/src/main/java/org/simpleyaml/configuration/file/YamlConfiguration.java#L101)

**Comment by Carleslc** [comment]
This is the intended behaviour since version [1.7.1](https://github.com/Carleslc/Simple-YAML/releases/tag/1.7.1), where list indentation is the same as [`YamlConfiguration.options().indent()`](https://carleslc.me/Simple-YAML/doc/Simple-Configuration/org/simpleyaml/configuration/ConfigurationOptions.html#indent(int)) (2 by default).

Maybe an `indentList` option can be added to [YamlConfigurationOptions](https://carleslc.me/Simple-YAML/doc/Simple-Yaml/org/simpleyaml/configuration/file/YamlConfigurationOptions.html) to allow this customization.

**Comment by Carleslc** [comment]
The default behaviour was not working properly, so the fix and the [`indentList`](https://carleslc.me/Simple-YAML/doc/org/simpleyaml/configuration/file/YamlConfigurationOptions.html#indentList-int-) option has been released in [1.7.3](https://github.com/Carleslc/Simple-YAML/releases/1.7.3). See #52 for further details.

Now your example is formatted by default as follows:

```yml
test-path:
  - '1': test1
    '2': test2
    '3': test3
  - '4': test1
    '5': test2
    '6': test3
  - '7': test1
    '8': test2
    '9': test3
```

You can remove the extra indent to match your example expected output setting `indentList` to 0.
